src/test/java/org/springframework/samples/petclinic/model/ValidatorTests.java

```diff
@@ -39,11 +39,20 @@ private Validator createValidator() {
 		return localValidatorFactoryBean;
 	}
 
+	@Test
+	void validateAge() {
```

Comment:
Bad test. Add test when you are setting age > 130 and < 18

Reply:
no-no
You should setAge(150) and catch exception. That is expected behaviour 
assertThat.throwExc smth like that